cmd_vel_bridge:
  ros__parameters:

    input_topic: /cmd_vel
    publish_twist: true
    publish_stamped: true
    output_topic_twist: /cmd_vel_outcast
    output_topic_stamped: /tricycle_controller/cmd_vel
    default_frame_id: ''
    print_messages: true